packages/modals/src/styled/StyledClose.ts

```diff
@@ -5,22 +5,42 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
+import styled, { css, DefaultTheme, ThemeProps } from 'styled-components';
+import rgba from 'polished/lib/color/rgba';
 import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'modals.close';
 
-export interface IStyledCloseProps {
-  hovered?: boolean;
-}
+const colorStyles = (props: ThemeProps<DefaultTheme>) => {
+  const baseColor = getColor('primaryHue' as string, 600, props.theme);
+
+  return css`
+    background-color: transparent;
+    color: ${getColor('neutralHue' as string, 600, props.theme)};
+
+    &:hover {
+      background-color: ${rgba(baseColor as string, 0.08)};
+      color: ${getColor('neutralHue' as string, 700, props.theme)};
+    }
+
+    &[data-garden-focus-visible] {
+      box-shadow: ${props.theme.shadows.md(rgba(baseColor as string, 0.35))};
+    }
+
+    &:active {
+      background-color: ${rgba(baseColor as string, 0.2)};
+      color: ${getColor('neutralHue' as string, 800, props.theme)};
```

Comment:
Why are these `as string` specifiers needed on `getColor`? I've not seen this in other areas of the code.

Reply:
Whoops, this was leftover from some earlier work, it's not needed and is removed 👍 .